spec/node-spec.js

```diff
@@ -462,6 +462,14 @@ describe('node feature', () => {
     it('should be able to create an aes-256-cfb cipher', () => {
       require('crypto').createCipheriv('aes-256-cfb', '0123456789abcdef0123456789abcdef', '0123456789abcdef')
     })
+
+    it('should list des-ede-cbc in getCiphers', () => {
+      expect(require('crypto').getCiphers()).to.include('des-ede-cbc')
+    })
+
+    it('should be able to create an des-ede-cbc cipher', () => {
+      require('crypto').createCipheriv('des-ede-cbc', '0123456789abcdeff1e0d3c2b5a49786', 'fedcba9876543210')
```

Comment:
I think the IV needs to be a different length here. try 8 or 32?
```suggestion
      require('crypto').createCipheriv('des-ede-cbc', '0123456789abcdeff1e0d3c2b5a49786', 'fedcba98')
```

Reply:
hmm: i pulled that key and IV from Chromium's own tests [here](https://cs.chromium.org/chromium/src/third_party/boringssl/src/crypto/cipher_extra/test/cipher_tests.txt?q=des-ede-cbc&sq=package:chromium&dr=C&l=94)

Reply:
huh, that is odd then. it'd be worth tracking down what exactly is going on & why node things this is the wrong IV length!

Reply:
@ryzokuken any ideas?

Reply:
I'm not really sure what Chromium's tests are doing there, but I'm looking at [the definition of `EVP_des_ede_cbc`](https://github.com/google/boringssl/blob/15ba2d11a9873cb40b2a45bba18242fdb16ca4c6/crypto/fipsmodule/cipher/e_des.c#L183-L193) and it looks like the IV and keys should be half the length this test sets them to:

```cpp
DEFINE_METHOD_FUNCTION(EVP_CIPHER, EVP_des_ede_cbc) {
  // ...
  out->key_len = 16;
  out->iv_len = 8;
  // ...
}
```

Reply:
oh, you know what it is? Chrome's tests encode the bytes as hex, which are 2 chars per byte.

Reply:
```suggestion
      require('crypto').createCipheriv('des-ede-cbc', Buffer.from('0123456789abcdeff1e0d3c2b5a49786', 'hex'), Buffer.from('fedcba9876543210', 'hex'))
```